### PR TITLE
Create ru.ts

### PR DESCRIPTION
**Pull request by vanja-san**

### CODE REVIEW

File: src/locale/ru.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  },
+
+  dialog: {
+    add_torrents: {
+      placeholder: 'Начните скачивать торренты, переместив их сюда,\nили нажмите кнопку вложения справа, чтобы выбрать.',
+      hint: 'Одна ссылка на строку',
+    },
+    delete_torrents: {
+      msg: 'Удалить выбранные торренты из списка передачи?',
+      also_delete_same_name_torrents: 'Также удалить один торрент с тем же именем |||| Также удалить %{smart_count} торрент(а\ов) с тем же именем',
```
Comment (CzBiX) [inline src/locale/ru.ts:97]: What's the purpose of `\o`? It's a illegal escape character in js string.

Comment (R-L-T-Y) [reply src/locale/ru.ts:97]: it's not `\o` tho

Comment (xumix) [reply src/locale/ru.ts:97]: > 
> 
> it's not `\o` tho

anway it should be `торрент(а/ов)`

**Comment by CzBiX** [comment]
@vanja-san Any updates?

**Pull request closed without merging**